rtdata/languages/default

```diff
@@ -1528,6 +1528,17 @@ HISTORY_MSG_WAVSTREND;Strength soft
 HISTORY_MSG_WAVTHRDEN;Threshold local contrast
 HISTORY_MSG_WAVTHREND;Threshold local contrast
 HISTORY_MSG_WAVUSHAMET;Clarity method
+HISTORY_MSG_WBALANCE_OBSERVER10;Observer 10°
+HISTORY_MSG_WBITC_SAMPLING;Low sampling
```

Comment:
Entries for removed GUI components should be removed too so translators don't have extra work :)